Sources/SwiftSvg/Svg+G.swift

```diff
@@ -0,0 +1,15 @@
+//
+//  File.swift
+//  
+//
+//  Created by Aaron Ge on 2022/4/26.
+//
+
+import SwiftSvg
+
+
+extension Svg{
+    open class G: Tag{
```

Comment:
Can you move this out from the Svg extension & follow the standard conventions? See Line, Rect, etc. tags. Also could you provide docs & test for these new tags? Thank you.

Reply:
move G Tag out from Svg extension